backend/src/tasks/search-sync.ts

```diff
@@ -4,8 +4,13 @@ import { In } from 'typeorm';
 import ESClient, { WebpageRecord } from './es-client';
 import S3Client from './s3-client';
 import PQueue from 'p-queue';
+import { chunk } from 'lodash';
```

Comment:
Do we have lodash installed? If not, perhaps we just do the chunking ourselves?

Reply:
We already have lodash installed.